### ISSUE TITLE
Complaint: Techknowlogick (Gitea)

### ISSUE THREAD

**Issue by Kreyren** [Kreyren/kreyren#107]
@lunny This is for a second time they (@techknowlogick) banned me without any explanation or rational reasoning when i didn't even interact with the repository for like 5 months atm..

First they claimed that i am in their words "disgusting racist" up until this day i have no idea what made them think that despite me being an activist who is very vocal for social equality and now whatever are they going to make up this time.

Take away their admin permissions already.

![tech_matrix_ban](https://user-images.githubusercontent.com/11302521/190848820-01525a1a-902d-4eac-9ac6-4325359a605d.png)

CC @6543 @zeripath

**Comment by Kreyren** [comment]
Matrix room: https://matrix.to/#/!bXIgKPtRdSXHBnVBaz:osgeo.org/$wamXRaekXzWNjeuWcpaW_apTi9CNkOx6Y1jDa3JJlmA?via=libera.chat&via=t2bot.io&via=matrix.org

TLDR: Was trying to report an issue observed on git.dotya.ml which is causing my system to halt when i press an UI button due to loading of lot of file changes
